libs/@blockprotocol/type-system/crate/src/ontology/data_type/mod.rs

```diff
@@ -206,7 +206,9 @@ mod tests {
                   "type": "string"
                 }
             ),
-            ParseDataTypeError::InvalidVersionedUrl(ParseVersionedUrlError::AdditionalEndContent),
+            ParseDataTypeError::InvalidVersionedUrl(ParseVersionedUrlError::InvalidVersion(
+                "invalid digit found in string".to_owned(),
```

Comment:
We should change this to something _like_ "version suffix wasn't a valid non-negative integer", ideally we'd also return what the invalid component was

Reply:
We cannot return the exact component but at least the string, which was failing to parse

Reply:
That's what I meant :) The component after `v/`

Reply:
Should be fixed